packages/client-react-streaming/src/DataTransportAbstraction/index.ts

```diff
@@ -7,7 +7,7 @@ export {
   useSuspenseQuery,
   useReadQuery,
   useBackgroundQuery,
-} from "./hooks.js";
+} from "@apollo/client/index.js";
```

Comment:
No more custom implementations & it doesn't matter where it's imported from 🎉

Reply:
I'm curious, is there any reason not to just deprecate these exports and have users import directly from `@apollo/client`? That could simplify things even further 🙂

Reply:
Removing it from this package, but keeping it in the "outer" package for now. (Will drop it in a future version.)